build.py

```diff
@@ -466,6 +470,7 @@ def extra_configs(dot_config, kbuild_output):
     bmeta['arch'] = "%s" %arch
     bmeta["cross_compile"] = "%s" %cross_compile
     bmeta["compiler_version"] = "%s" %gcc_version
+    bmeta["build_environment"] = "%s" % compiler_image
```

Comment:
This is true for every field that's read from the environment. Build.py doesn't check anything.

Sure it would be nice to fix that but it's not making the situation any worse.

Reply:
OK so that's something to add to the "clean-up build.py" list.